frog/imports/ui/graph/DraggableAc.jsx

```diff
@@ -0,0 +1,215 @@
+import React, { Component, PropTypes } from 'react';
+import Rnd from 'react-rnd';
+
+import { updateGraphActivityDuration } from '../../api/activities'
+import { computeTopPositionFromGraph, convertTimeToPx,
+          convertPxToTime, textSizeAndMargin, horizontalZoom } from './graph_utils.js'
+
+const boxHeight = 40
+
+const defaultTime = 10
+
+const circleRadius = 6
+
+const rndMargin = 12
+
+const minRealBox = 45
+// #fe8181 ligth red
+const divStyle = (duration) =>
```

Comment:
Since a month we want to do it, but everytime there was something else to do, so that after being reported everytime it was never done.